APKBUILD

```diff
@@ -25,8 +25,6 @@ package() {
   rm -rf "$pkgdir"/usr/glibc-compat/lib/gconv
   rm -rf "$pkgdir"/usr/glibc-compat/lib/getconf
   rm -rf "$pkgdir"/usr/glibc-compat/lib/audit
-  rm -rf "$pkgdir"/usr/glibc-compat/lib/*.a
-  rm -rf "$pkgdir"/usr/glibc-compat/include
```

Comment:
This will include them in the `glibc-compat` package. I think you want to keep these two lines here and instead create a `dev() {}` function that adds these two (see the `bin` function example which is run for the `glibc-compat-bin` subpackage).